tests/functional/test_to_service_interstitial.py

```diff
@@ -0,0 +1,39 @@
+import re
+
+from bs4 import BeautifulSoup
+from flask import url_for
+import pytest
+
+
+def request(url, method):
+    r = method(url)
+    r.soup = BeautifulSoup(r.get_data(as_text=True), 'html.parser')
+    return r
+
+
+class When_on_to_service_interstitial(object):
+
+    @pytest.fixture(autouse=True)
+    def setup_page(self, client, app):
+        self.config_countdown = app.config['META_REFRESH_DELAY']
+
+        self.service_page_redirect = 'to-service-page'
+
+        with client.session_transaction() as session:
+            session['auth_redirect'] = self.service_page_redirect
+
+        self.response = request(url_for('main.to_service'), client.get)
+
+    def it_has_continue_link_href_set_to_service_page(
+            self, client):
+        next_li = self.response.soup.find("li", class_="next")
+
+        assert self.service_page_redirect == next_li.find('a')['href']
+
+    def it_has_countdown_set_to_the_meta_refresh_config(self, config):
+        countdown_text = self.response.soup.select_one(
+            "#content span.pagination-text span").text
```

Comment:
This CSS selector is both too specific (`#content` is unnecessary and `span.pagination-text` doesn't need to specify `span.`) and not specific enough (without an `id` or `class` the last `span` is not specific enough, and requires the previous selectors to provide context).

This should be
```python
countdown_text = self.response.soup.find('.redirect-timer').text`
```

Also, `redirect_timer_text` might be a better variable name, since the word "countdown" does not appear anywhere except in this test.